### ISSUE TITLE
Not working after lsusb identifier changes?

### ISSUE THREAD

**Issue by LillyKane** [benthor/HuionKamvasGT191LinuxDriver#3]
I have gone through the setup (several times now) and have run into a frustrating problem. When I plug in my tablet and run lsusb I see the entry 
`Bus 001 Device 005: ID 0416:a316 Winbond Electronics Corp.`
I am able to run `sudo uclogic-probe 1 5` here and get the output
`M 57 00 50 00 4D 00 20 00 55 00 53 00 42 00
P 57 00 50 00 4D 00 20 00 55 00 53 00 42 00`
After about three seconds of being plugged in though the entry is missing if I run lsusb again, replaced by `Bus 001 Device 006: ID 256c:0006e`. The ID is always consistent. Running uclogic-probe on this device gives an error: `Failed to get manufacturer string descriptor: Pipe error (ERROR_PIPE)`. Is this preventing kamvas.py from running properly considering issue 1?

**Comment by doakey3** [comment]
I am also having this issue. If I run the following .sh file within 3 seconds of plugging in the graphics tablet: 
```
T="Winbond Electronics Corp."
BUS=$(lsusb | grep "$T" | sed -e 's|Bus \([0-9]*\) Device \([0-9]*\):.*$|\1|g')
DEV=$(lsusb | grep "$T" | sed -e 's|Bus \([0-9]*\) Device \([0-9]*\):.*$|\2|g')

sudo rmmod hid_uclogic
sudo modprobe uinput
sudo uclogic-probe $BUS $DEV | uclogic-decode
```
the output I get is 
```
  Manufacturer: ????????????
       Product: NUVOTON
```
Running kamvas.py after doing this will not work.

**Comment by SHARKvince** [comment]
Yeah, I ran into the same issue unfortunately. Not sure what to do.

**Comment by wrightwriter** [comment]
Same issue. Any fix ? Works on Windows.

**Comment by wrightwriter** [comment]
> T="Winbond Electronics Corp." BUS=$(lsusb | grep "$T" | sed -e 's|Bus \([0-9]*\) Device \([0-9]*\):.*$|\1|g') DEV=$(lsusb | grep "$T" | sed -e 's|Bus \([0-9]*\) Device \([0-9]*\):.*$|\2|g') sudo rmmod hid_uclogic sudo modprobe uinput sudo uclogic-probe $BUS $DEV | uclogic-decode

I have exactly the same problem. Have you guys found a way to fix this ?

**Comment by joseluis** [comment]
Hi @wrightwriter could you try my [huion-linux-drivers](https://github.com/joseluis/huion-linux-drivers) repo and see if it works for you?

**Comment by wrightwriter** [comment]
> Hi @wrightwriter could you try my [huion-linux-drivers](https://github.com/joseluis/huion-linux-drivers) repo and see if it works for you?

Hey, sorry I took so long, I tried them and wrote up the results in your repo.

**Comment by wrightwriter** [comment]
> Hi @wrightwriter could you try my [huion-linux-drivers](https://github.com/joseluis/huion-linux-drivers) repo and see if it works for you?

Hey, it starts up, but I have not been able to find my pen-charging cable for the past 3 months, so I can't check out if  it works. : \
